packages/bot-web-ui/src/utils/__tests__/window-size.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import {
+    getMainContentHeight,
+    getMainContentWidth,
+    setMainContentWidth,
+    setInnerHeightToVariable,
+} from '../window-size';
+
+describe('DOM utilities of window-size file', () => {
+    beforeEach(() => {
+        render(<div className='bot' data-testid='mock-bot' />);
+    });
+
+    it('Should get main content height', () => {
+        const bot_element = screen.getByTestId('mock-bot');
+
+        if (!bot_element) {
+            throw new Error("Element with class '.bot' was not found");
+        }
```

Comment:
@maryia-matskevich-deriv is it necessary to have this error here? I think the element will always be there as we are only rendering the mock so this line may not be necessary :thought_balloon:

Reply:
done✅